shoplist/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="com.netguru.codereview.shoplist" />
```

Comment:
You tried to move the manifest settings to the module shoplist but maybe forget to add it in the changes file.